voila/handler.py

```diff
@@ -140,7 +140,8 @@ def _jinja_cell_generator(self, nb, kernel_id):
             result = executenb(nb, km=km, cwd=self.cwd, config=self.traitlet_config)
             cell = result.cells[0]  # keep a reference to the executed cell
             nb.cells = all_cells  # restore notebook in case we access it from the template
-            # we don't filter empty cells, since we do not know how many empty code cells we will have
+            # We don't filter empty cells, since we do not know how many empty code cells we will have
+            # We'll do the filtering in CSS
```

Comment:
I think we can/should now remove all reference to stripping these cells in the code.

Reply:
I disagree. We strip cells in the `_jinja_notebook_execute` method, but not in `_jinja_cell_generator` method, which is pretty heterogeneous so this comment is important.

Also, I think it should not be fixed in CSS but in Python. Only that we did not find a way to do that correctly...